src/openfermion/utils/_linear_qubit_operator_test.py

```diff
@@ -187,6 +179,14 @@ def test_matvec_compare(self):
                          for v in numpy.identity(16)])),
                                        mat_expected.A))
 
+    def test_closed_workers_not_resued(self):
+        """Issue #461 - no longer attempting to reuse closed worker pool"""
+        qubit_op = QubitOperator('X0 Z5')
+        linear_qubit_op = generate_linear_qubit_operator(
+            qubit_op,
+            options=LinearQubitOperatorOptions(processes=2))
+        get_ground_state(linear_qubit_op)
+
```

Comment:
Remove this test. Instead, add a test to ParallelLinearQubitOperatorTest that simply constructs a ParallelLinearQubitOperator and applies it twice to the same state (similar to what I did in the issue). No need to mention the issue in the docstring.